harbor-mixin/dashboards/harbor-overview.json

```diff
@@ -0,0 +1,2485 @@
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 3,
+        "w": 4,
+        "x": 0,
+        "y": 3
+      },
+      "id": 9,
+      "options": {
+        "cellHeight": "lg",
+        "footer": {
+          "countRows": false,
+          "enablePagination": false,
+          "fields": "",
+          "reducer": [
+            "sum"
+          ],
+          "show": false
+        },
+        "showHeader": false,
+        "sortBy": [
+          {
+            "desc": false,
+            "displayName": "Harbor Version"
+          }
+        ]
+      },
+      "pluginVersion": "10.0.2-cloud.1.94a6f396",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "$datasource"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "max(harbor_system_info{job=~\"$job\", cluster=~\"$cluster\"}) by (harbor_version)",
+          "format": "table",
+          "legendFormat": "",
+          "range": true,
+          "refId": "A"
+        }
+      ],
+      "title": "Harbor Version",
+      "transformations": [
+        {
+          "id": "organize",
+          "options": {
+            "excludeByName": {
+              "Time": true,
+              "Value": true
+            },
+            "indexByName": {},
+            "renameByName": {
+              "harbor_version": "Harbor Version"
+            }
+          }
+        }
+      ],
+      "transparent": true,
+      "type": "table"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "$datasource"
+      },
+      "description": "Total number of public and private projects",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "continuous-BlPu"
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "none"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 4,
+        "w": 4,
+        "x": 0,
+        "y": 6
+      },
+      "id": 1,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "textMode": "auto"
+      },
+      "pluginVersion": "10.0.2-cloud.1.94a6f396",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "$datasource"
+          },
+          "editorMode": "code",
+          "expr": "sum(harbor_project_total{job=~\"$job\", cluster=~\"$cluster\"})",
```

Comment:
Is sum aggregating away a label here? Would it be useful to have a template variable that allowed you to filter on that/those labels instead?

Reply:
Sorry, I dont follow, Could you share an example

Reply:
More of a broad question about the metrics, which I am not super familiar with. Are there other labels besides `job` and `cluster` on `harbor_project_total`?

For example is there a `namespace` label (total SWAG, probably a bad example) label which it would be useful to group by for this?

Likely not, but, just checking to see if there's any useful labels/dimensions which are getting aggregated away when you `sum`

Reply:
just the `public` label which mentions if the project is public or private

Reply:
Okay, that *might* be a useful template variable, but I'll leave it to your discretion. You could also use some transforms to display public and private separately there, or just add queries which are explicit. I.E. `sum(harbor_project_total{..., public=m|true})` with a legend of `Public`.

If there isn't any significant value in differentiating, you can just leave this be.

Reply:
yeah, my thoughts kinda divided on this,  Thinking of keeping it as is for now and later see if there are any particular requests for it and update.